### ISSUE TITLE
How to prevent other type of file upload

### ISSUE THREAD

**Issue by iftekharul-dev** [yudax42/vue-upload-drop-images#18]
I want to prevent to upload other types of file. but its uploading all types of file .

**Comment by yudax42** [comment]
hello @iftekharul-dev,

It's already accepting just the files of type image as you see here: 
https://github.com/yudax42/vue-upload-drop-images/blob/c1e6837560b7075cc99a55cadc460769cc687f74/dist/vue-upload-drop-images.vue#L123

Please explain more your issue so I can help you. 

Thank you!

**Comment by iftekharul-dev** [comment]
Yes, i saw that, but its accept all files.Also not working on your demo [https://vueupload.yudax.dev/](url) :( ..

**Comment by yudax42** [comment]
Can you please tell me which browser are you using?
Currently, it's using just `accept` input which works only on this list of browsers:

- Chrome 16 +
- Safari 6 +
- Firefox 9 +
- IE 10 +
- Opera 11 +

I will try to validate the input files using javascript in the future.

**Comment by iftekharul-dev** [comment]
Its Chrome 16 +

**Comment by CyferZ** [comment]
Interesting, on Firefox 9+ it does the same. @iftekharul-dev I suggest just adding validation to handle this until @yudax42 finds something.

**Comment by isaacssemugenyi** [comment]
Can this be open for one to set what images they accept other than accepting all image types.
